### ISSUE TITLE
Can't seem to draw a circle using `Draw`

### ISSUE THREAD

**Issue by amirrajan** [HipByte/motion-game#7]
Here is the draw code I've added to `MainScene < MG::Scene`

```
circle = MG::Draw.new
white = MG::Color.new
white.red = 1.0
white.green = 1.0
white.blue = 1.0
circle.dot([0, 0], 50, white)
add circle
```

From http://discuss.cocos2d-x.org/t/how-to-to-draw-circle-cocos2dx3-0/17218

```
DrawNode *drawnode = DrawNode::create();
drawnode->drawDot(ccp(15,15), 50, Color4F(1,1,1,1));
this->addChild(drawnode);
```

Aside: It doesn't look like `Draw` and `DrawNode` have any touch events, correct?

**Comment by jjaffeux** [comment]
@amirrajan hi

if you check your `white` locale you will see a `Color` object where `alpha=0`.

Using this code should fix your issue:

```ruby
    white = MG::Color.new
    white.red = 1.0
    white.green = 1.0
    white.blue = 1.0
    white.alpha = 1.0

    circle = MG::Draw.new
    circle.dot([50, 50], 50, white)
    add circle
end
```

I understand this is quite unexpected, maybe @lrz has an opinion on this.

**Comment by lrz** [comment]
Yeah by default all properties of a Color object are set to 0.

Also, I believe you can pass the `:white` symbol instead of creating this Color object.

```
circle.dot(..., :white)
```

**Comment by jjaffeux** [comment]
@lrz My question was shouldn't we have rgb set to 0 and alpha to 1 as defaults ? I understand amir didn't see that alpha was at 0, it's totally unexpected. Also, the documentation doesn't say anything about the 0 default.

**Comment by lrz** [comment]
I see, you make a point, I think we should indeed set alpha to 1 by default. I will fix that and edit the documentation.

**Comment by lrz** [comment]
Fixed as of 2cd0f46825f84683150f8aa011f706929d7f17c8
